utilities/download_old_firefox.py

```diff
@@ -9,43 +9,19 @@
     today = date.today()
     # Set correct month for URL builder
     if len(f"{today.month}") < 2:
-        current_month = f"0{today.month}"
+        download_month = f"0{today.month - 1}"
```

Comment:
I think I'm confused here, but not sure why (apart from my own staggering incompetence).
Trying to mentally run all the combinations of dates to see where my bounds are.

| DATE | EXPR | true/false | STR | VALUE |
|----|----|:----:|----|----:|
| November 14 | `len(f"{11}") < 2` | falsey | `f"{11 - 1}"` | "10"
| October 14 |  `len(f"{10}") < 2` | falsey | `f"{10 - 1}"` | "9"
| September 14 | `len(f"{9}") < 2` | truthy | `f"0{9 - 1}"` | "08"
| January 14 | `len(f"{1}") < 2` | truthy | `f"0{1 - 1}"` | "00"

Reply:
I found the same, january looks like it would fail, so I should add a specific case for it

Reply:
but October fails too, right?
Because in October `today.month` is "10" which passes the `len("10") < 2` check but then once you subtract a month, it becomes September and doesn't zero-pad so might leave at "9" instead of "09".